src/materials/minetest.lua

```diff
@@ -60,23 +68,41 @@ local materials = {
     stone = "default:stone",
     desert_stone = "default:desert_stone",
     desert_sand = "default:desert_sand",
+    desert_sandstone = "default:desert_sandstone",
+    silver_sand = "default:silver_sand",
+    silver_sandstone = "default:silver_sandstone",
     chest = "default:chest",
     cobble = "default:cobble",
+    mossycobble = "default:mossycobble",
     brick = "default:brick",
+    obsidian = "default:obsidian",
     obsidian_glass = "default:obsidian_glass",
     water_source = "default:water_source",
     water_flowing = "default:water_flowing",
+    river_water_source = "default:river_water_source",
+    river_water_flowing = "default:river_water_flowing",
+    lava_source = "default:lava_source",
+    lava_flowing = "default:lava_flowing",
     dirt = "default:dirt",
     dirt_with_grass = "default:dirt_with_grass",
+    dirt_with_snow = "default:dirt_with_snow",
+    snowblock = "default:snowblock",
+    ice = "default:ice",
     apple_leaves = "default:leaves",
     apple_log = "default:tree",
     apple_planks = "default:wood",
+    acacia_planks = "default:acacia_wood",
     birch_leaves = "default:aspen_leaves",
     birch_log = "default:aspen_tree",
     birch_planks = "default:aspen_wood",
     jungle_leaves = "default:jungleleaves",
     bowl = "",
     stick = "default:stick",
+    wheat = "farming:wheat",
+    wheat_seed = "farming:seed_wheat",
+    dry_shrub = "default:dry_shrub",
+    insulation = "mesecons_materials:fiber",
```

Comment:
shouldn't this be dependent on mesecon_materials being available?

Reply:
this is a draft, not ready for review